docs/source/kinds_of_types.rst

```diff
@@ -802,10 +810,10 @@ item types:
 
     Point = NamedTuple('Point', [('x', int),
                                  ('y', int)])
-    p = Point(x=1, y='x')  # Argument has incompatible type "str"; expected "int"
+    p = Point(x=1, y='x')  # error: Argument 2 to "Point" has incompatible type "str"; expected "int"
 
-Python 3.6 will have an alternative, class-based syntax for named tuples with types.
-Mypy supports it already:
+Python 3.6 has an alternative, class-based syntax for named tuples with types,
+which Mypy supports:
```

Comment:
I don't think we need to say `which Mypy supports` -- that is implied.

It was not obvious before Python 3.6 was released but now it is.